EIPS/eip-draft_deposit_contract_snapshot_interface.md

```diff
@@ -0,0 +1,282 @@
+---
+eip: <to be assigned>
```

Comment:
```suggestion
eip: 4881
```

Reply:
Thank you! I've made this change in my latest commit.